### DISCUSSION TITLE
"locating pins" on Create3 as structural anchors

### DISCUSSION THREAD

**Discussion by gruedisueli** [iRobotEducation/create3_docs#364] [Questions] [answered]
I'm working on creating a top plate for the Create3 that has more resistance to rotational forces and won't pop off if the robot turns while pushing on a modestly heavy load above it. The build plate that comes with the robot is great but probably not the best for my task.

I'm thinking of tying my new top plate down to the screw holes for the "locating pins" as noted in your [documentation](https://iroboteducation.github.io/create3_docs/hw/overview/). 

A couple questions: 
-Are the locating pins bolted into anything important inside the iRobot (electronics, etc)?
-Can you provide the specs for the type of screw these are (diameter and thread spacing).

Thanks!

**Comment by shamlian** [comment, marked as answer]
Why not drop an M3 or #4 screw in the [hole toward the rear of the robot](https://iroboteducation.github.io/create3_docs/hw/mechanical/#locking-the-faceplate) to lock the faceplate in place?

The screws securing the locating pin screw into plastic bosses designed for self-threading plastic screws. They are roughly M3 or #4; since it's a plastic boss for a self-threading screw, there's some flexibility, but also because it's a self-threading screw, you need to be careful not to destroy the boss. (These sorts of screws and screw bosses are not _really_ designed for re-threading.)

The bosses toward the rear of the robot go clear through the top of the robot into the cargo area. The bosses toward the front of the robot get close to things you don't want to put a hole through, so please be careful if you are going to put a longer screw in.

**Reply by gruedisueli** [reply]
@shamlian Thank you. This is super helpful. I'll avoid using the pins for this reason. I did see the rear hole before--it's a very thoughtful inclusion in the design of the product! Originally, I was a bit worried about shearing of the plastic if we turn the robot and whatever is on top has a lot of inertia. We aren't putting any vertical loads on the robot, as we have an external frame with casters that is riding on top, but the robot is acting as pusher/rotator for the fabrication that will be above it, so I've been trying to mind the kinds of forces this would put on the connections at the top of the robot. 

I think what I'll do is use the rear pin mainly to hold the printed top plate I'm making in place, but rely on other registrations (existing wells in the top of the robot) to resist turning and pushing forces.

Thanks!
